### ISSUE TITLE
Add Korporatio to Incorporations Marketplace

### ISSUE THREAD

**Issue by CortecKYC** [SelfKeyFoundation/Identity-Wallet#1771]
Once their KYCC instance is up and running in production, add Korporatio to Incorporations Marketplace as the exclusive vendor for Seychelles.
- [x] It doesn't require any kind of highlight (confirmed with Kim)
- [x] Remove Flag Theory's offer for Seychelles

**Comment by rhdelima** [comment]
@sk91 @andregoncalves I don't see Seychelles on the Incorporation Marketplace. Is this an airtable issue?
Build: dev_2019.12.19.11

**Comment by andregoncalves** [comment]
What env variables are you using?

**Comment by rhdelima** [comment]
@andregoncalves I'm using these:
1. MATOMO_SITE=2 ATTRIBUTE_TYPE_SOURCE_OVERRIDE='development' FEATURE_CORPORATE_MARKETPLACE=1
2. MATOMO_SITE=2 only
Can you share what env variables to use?

**Comment by andregoncalves** [comment]
`FORCE_DEV_MODE=1` should work. 
Korporatio is not active yet in prod airtable.

**Comment by rhdelima** [comment]
@andregoncalves 
Please see issues below:
1. Error when submitting requirements
![image](https://user-images.githubusercontent.com/10294014/71941653-82d58000-31f5-11ea-904f-ad9eac9e73ca.png)
![image](https://user-images.githubusercontent.com/10294014/71941672-9385f600-31f5-11ea-8ad3-47cef4efa710.png)

2. Missing link for `terms and conditions`
![image](https://user-images.githubusercontent.com/10294014/71941494-e3b08880-31f4-11ea-9d6c-aea1f0294568.png)

**Comment by CortecKYC** [comment]
@andregoncalves ^^

**Comment by andregoncalves** [comment]
There's a PR for fixing the missing terms and conditions link.
The issue with the submission is related to the test instance (not the same instance used in production).
This doesn't block the release but it should be fixed, so until that happens this issue should not be closed.

**Comment by andregoncalves** [comment]
@rhdelima you can retest now, the KYCC test instance for Korporatio should be working now.
Let me know if you need access credentials to the instance.

**Comment by rhdelima** [comment]
@andregoncalves I submitted an application but I don't see it on the korporatio test instance.
`MATOMO_SITE=2 FORCE_DEV_MODE=1 INCORPORATIONS_PRICE_OVERRIDE=0 PRIMARY_TOKEN=KI CHAIN_ID=3 DEV_TOOLS=yes open SelfKey\ Identity\ Wallet.app`
![image](https://user-images.githubusercontent.com/10294014/72408917-3f53b680-379f-11ea-8dca-44e7b94bc5b2.png)
![image](https://user-images.githubusercontent.com/10294014/72409036-978ab880-379f-11ea-8bcd-4f7dec27bc8c.png)

**Comment by andregoncalves** [comment]
@rhdelima That's not the korporatio test instance. that's their *production* instance!
Test instance is: https://selfkey-dev.instance.kyc-chain.com/management/

**Comment by rhdelima** [comment]
@andregoncalves my bad. Can you share the credentials, please?

**Comment by andregoncalves** [comment]
@rhdelima shared on mattermost.

**Comment by rhdelima** [comment]
@andregoncalves 
Please see my observations:
1. I think passport and selfie should be separated in the template. 
2. Also, unable to view passport attachments. https://recordit.co/GoA69lW4i6

**Comment by andregoncalves** [comment]
1. We have been discussing that change - but currently not possible
2. This seems to be KYCC problem - @CortecKYC who should I ping on kycc team to investigate?

**Comment by CortecKYC** [comment]
@andregoncalves Yes, please

**Comment by susananakatani** [comment]
> @andregoncalves
> Please see my observations:
> 
> 1. I think passport and selfie should be separated in the template.
> 2. Also, unable to view passport attachments. https://recordit.co/GoA69lW4i6

@andregoncalves do you mean that KYCC instances integrated in the marketplace ( like Korporation) are not using the ID Verification tool for processes started from the marketplace?

**Comment by CortecKYC** [comment]
This ticket is on hold. KYCC devs are working on a temporary solution until v5 is rolled out (v5 would solve this issue but it's not ready yet).

**Comment by CortecKYC** [comment]
Followed up on Mattermost: https://mattermost.kyc-chain.com/kyc-chain/pl/t1ie55b16fb9ze5tehppcbs5ro

tl;dr KYCC v5 is not ready yet.

**Comment by CortecKYC** [comment]
Korporatio was added to the Marketplace. This ticket is still on QA because of the KYCC passport/selfie issue.
